### DISCUSSION TITLE
My profile is ugly

### DISCUSSION THREAD

**Discussion by seon22break** [community/community#45222] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to make a profile with this styles? 
![imagen](https://user-images.githubusercontent.com/36485771/214318670-aaee5662-494b-4d4d-9ae8-9523bfce3c85.png)

**Comment by diegomarty** [comment, marked as answer]
Of course! You can do it very easily, there are many websites to create badges, for example, I use [shields.io](https://shields.io/) which is very useful and versatile. You can use it both in the profile and in your repositories!.

**Reply by seon22break** [reply]
Oh Thanks!
